app/sentry.ts

```diff
@@ -0,0 +1,55 @@
+import * as Sentry from '@sentry/browser';
+import { CaptureConsole, Dedupe, Ember } from '@sentry/integrations';
+import config from 'open-event-frontend/config/environment';
+
+Sentry.init({
+    integrations: [
+        new Ember(),
+        new Dedupe(),
+        new CaptureConsole({
+            levels: ['error']
+        })
+    ],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5114297)